cmd/lifecycle/rebaser.go

```diff
@@ -128,7 +136,7 @@ func (r *rebaseCmd) Exec() error {
 	} else {
 		newBaseImage, err = remote.NewImage(
 			r.imageNames[0],
```

Comment:
```suggestion
			r.runImageRef,
```
Unrelated bug I think.

Reply:
same here https://github.com/buildpacks/lifecycle/pull/409/files#diff-92c6ed289675df1e2edf598694dfc99f22a591496c130d10a43e86830cbd3686R193

Reply:
@ekcasey I'm having trouble following your link - could you show us the other line of code that should be changed? Also, just trying to follow what the code is doing here - what is the impact of this bug?

Reply:
https://github.com/buildpacks/lifecycle/pull/409/files#diff-b19694987281b87997e43d8b402d163180ec5b9763cd3567079168459c0fec56R97-R107

re impact: small but if there was ever an error getting data from the newBaseImage, the image would return a misleading error that includes the wrong image name. E.g if https://github.com/buildpacks/lifecycle/blob/main/rebaser.go#L57 errored you would get a bad error string here https://github.com/buildpacks/imgutil/blob/main/remote/remote.go#L390